app/1.0/blog.yaml

```diff
@@ -1,7 +1,15 @@
 ---
+- title: "Polymer News Progressive Web App"
+  path: /1.0/blog/2017-02-06-news-pwa
+  published: 2017-02-06
+  author: polymer-team
+  description: "The new Polymer News demo progressive web app helps you start building a cutting-edge PWA using Polymer."
+  tag:
+    - blog
+    - announcements
 - title: "webcomponents.org"
-  path: /1.0/blog/2017-01-09-webcomponents-org
-  published: 2017-01-09
+  path: /1.0/blog/2017-01-10-webcomponents-org
+  published: 2017-01-10
```

Comment:
Since this is already published with the -09- URL, I don't think we should move it at this point. (We can update the publication date in the YAML, but the path should stay the same.)